src/linux/init/util.cpp

```diff
@@ -1634,6 +1634,19 @@ Return Value:
     return 0;
 }
 
+int UtilMountFile(const char* Source, const char* Destination)
+try
+{
+    wil::unique_fd Fd{open(Destination, (O_CREAT | O_WRONLY | O_TRUNC), 0755)};
```

Comment:
Do we want O_TRUNC? Won't that clobber the file that's there if it currently exists?

Reply:
Hmm probably not. Although we're mounting over the file so the expectation would be for that file to be empty. We can probably remove O_TRUNC though

Reply:
IMO bind mounting over an existing file should be non-destructive.